app/routes/photos/components/GalleryDetail.tsx

```diff
@@ -94,62 +115,45 @@ const GalleryDetail = () => {
     setUpload(!upload);
   };
 
-  const navigate = useNavigate();
-
-  const handleClick = (picture) => {
+  const handleClick = (picture: GalleryListPicture) => {
     navigate(`/photos/${gallery.id}/picture/${picture.id}`);
   };
 
   const downloadGallery = () => {
     setDownloading(true);
     // Force re-fetch to avoid expired image urls
-    clear(gallery.id);
+    dispatch(clearGallery(gallery.id));
 
     const finishDownload = () => setDownloading(false);
 
     downloadNext(0, [])
       .then((blobs) => {
-        const names = pictures.map((picture) => picture.file.split('/').pop());
+        console.log(blobs);
```

Comment:
Whops also missed this one the first time around

Reply:
😅